brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3455 Fix: Global - Resolved an issue involving a block encounter error when attempting to select typography font families, stemming from the presence of multiple iframes on the page.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3455

### Screenshots

- https://d.pr/v/x16HKg

### Types of changes




- Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?

- Drag and drop Google map.
- After that use heading block.
- Go to style, Typography and select 2 or 3 fonts family
- Make sure you did not get block encounter an error issue.
- Switch to responsive devices and select typography font family again.
- Make sure you did not get block encounter an error issue

### Checklist:
